lambda/fulfillment/lib/middleware/1_parse.js

```diff
@@ -95,9 +95,13 @@ function getClientType(req) {
     }
     // Try to determine which Lex client is being used based on patterns in the req - best effort attempt.
     const voiceortext = (req._preferredResponseType == 'SSML') ? "Voice" : "Text" ;
-    if (_.get(req,"_event.requestAttributes.x-amz-lex:channel-type") == "Slack") {
+
+    //for LexV1 channels -- check for x-amz-lex:channel-type requestAttribute
+    //for LexV2 channels -- check for x-amz-lex:channels:platform requestAttribute
+
```

Comment:
Thanks for adding the comment line. Additionally, it may be useful to add to the comment with refer to lex documentation for more details:
- https://docs.aws.amazon.com/lex/latest/dg/example1.html
- https://docs.aws.amazon.com/lexv2/latest/dg/deploying-messaging-platform.html

Reply:
updated comments and included reference to Amazon Lex documentation.